addons/rating/models/mail_thread.py

```diff
@@ -21,5 +21,6 @@ def message_post(self, **kwargs):
                 'message_id': message.id,
                 'consumed': True,
                 'partner_id': self.env.user.partner_id.id,
+                'website_published': message.website_published,
```

Comment:
This is not the correct way of doing it. Indeed website published is defined in website mail / website rating. You will have to keep something modular :) .